README.md

```diff
@@ -1,761 +1,24 @@
-DevExtreme includes the [data layer](https://js.devexpress.com/Documentation/Guide/Data_Layer/Data_Layer/) that enables you to read and write data stored in different data sources.
-
-The example below demonstrates how to use a [DataSource](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/) with DevExtreme Components.
-
-```jsx
-import React from 'react';
-import ReactDOM from 'react-dom';
-
-import DataSource from 'devextreme/data/data_source';
-import List from 'devextreme-react/list';
-
-import 'devextreme/dist/css/dx.common.css';
-import 'devextreme/dist/css/dx.light.compact.css';
-
-const items = [
-    { text: '123' },
-    { text: '234' },
-    { text: '567' }
-];
-
-class Example extends React.Component {
-
-    constructor(props) {
-        super(props);
-
-        this.dataSource = new DataSource({
-            store: {
-                type: 'array',
-                data: items
-            },
-            sort: [
-                { getter: 'text', desc: true }
-            ],
-            pageSize: 1
-        });
-    }
-
-    render() {
-        return (
-            <List dataSource={this.dataSource} />
-        );
-    }
-
-    componentWillUnmount() {
-        this.dataSource.dispose();
-    }
-}
-
-ReactDOM.render(
-    <Example />,
-    document.getElementById('root')
-);
-```
-
-Note that a DataSource is considered as a 'service'. So, modifying its properties does not cause component rerendering.
-
-
-## <a name="typescript-support"></a>Typescript Support ##
-
-We provide TypeScript declarations for DevExtreme Components. Strict typing allows you to catch many bugs and improve your workflow by adding features like auto-completion and automated refactoring.
-
-Below is an example of appearance customization using TypeScript:
-
-```ts
-import * as React from "react";
-import * as ReactDOM from "react-dom";
-import List from "devextreme-react/list";
-
-import "devextreme/dist/css/dx.common.css";
-import "devextreme/dist/css/dx.light.compact.css";
-
-interface IListItemProps {
-    text: string;
-}
-
-class Item extends React.Component<IListItemProps, { counter: number }> {
-
-    constructor(props: IListItemProps) {
-        super(props);
-        this.state = {
-            counter: 0
-        };
-
-        this.handleClick = this.handleClick.bind(this);
-    }
-
-    public render() {
-        return (
-            <i onClick={this.handleClick}>
-                Component template for item {this.props.text}. <b>Clicks: {this.state.counter}</b>
-            </i>
-        );
-    }
-
-    private handleClick() {
-        this.setState({
-            counter: this.state.counter + 1
-        });
-    }
-}
-
-const items: IListItemProps[] = [
-    { text: "123" },
-    { text: "234" },
-    { text: "567" }
-];
-
-ReactDOM.render(
-    <List items={items} itemComponent={Item} />,
-    document.getElementById("root")
-);
-```
-
+* [Support & Feedback](#support-feedback)
```

Comment:
As the page is now very short I suggest removing the point from the list (as well as License)